### PR TITLE
enable cgo for backplane-2.2

### PR DESCRIPTION
**Pull request by zhujian7**

### CODE REVIEW

**Comment by zhujian7** [comment]
cc @elgnay

File: Makefile [hunk 1]
```diff
@@ -60,9 +60,13 @@ test: manifests generate fmt vet ## Run tests.
 
 ##@ Build
 
-build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/manager/main.go
-	go build -o bin/agent cmd/agent/main.go
+build: generate fmt vet build-manager build-agent
+
+build-manager:
+	CGO_ENABLED=1 go build -a -o bin/manager cmd/manager/main.go
```
Comment (elgnay) [inline Makefile:66]: Do we need  GOFLAGS="" as well?

File: Makefile [hunk 2]
```diff
@@ -60,9 +60,13 @@ test: manifests generate fmt vet ## Run tests.
...
+
+build-agent:
+	CGO_ENABLED=1 go build -a -o bin/agent cmd/agent/main.go
```
Comment (elgnay) [inline Makefile:69]: Do we need  GOFLAGS="" as well? It was set in the Dockerfile.

**Pull request merged**